### DISCUSSION TITLE
Is this project active? is it still being maintained?

### DISCUSSION THREAD

**Discussion by ghost** [servo/servo#28985] [Q&A] [answered]
Hi everyone.

I found the proposal of this open source repository interesting, but I would like to know... *Is this project active? is it still being maintained?*

**Comment by jdm** [comment, marked as answer]
The activity level is relatively low at the moment, but there are discussions happening about bringing on additional contributors.

**Reply by GPHemsley** [reply]
What is the relationship to the codebase used in mozilla-central? It seems a lot of active development is happening there that is not present in this repo?

**Reply by jdm** [reply]
The codebases (particularly the `style` crate) forked off of this repository several years ago, and stopped submitting changes some time after 2020.

**Comment by simonsan** [comment]
> Hi everyone.
> 
> I found the proposal of this open source repository interesting, but I would like to know... _Is this project active? is it still being maintained?_

The answer is yes: https://servo.org/blog/2023/02/03/servo-2023-roadmap/
